docs/src/main/paradox/user/lang/kotlin/gettingstarted.md

```diff
@@ -229,6 +242,58 @@ A minimal Maven example pom file, which uses the [Xolstice Maven Protocol Buffer
 ```
 @@@
 
+Gradle Kts
```

Comment:
@sleipnir Perhaps put this in an external file and then include it in the docs via tags?

Reply:
Thanks @viktorklang
I understand that it doesn't change much and I wouldn't have to worry about that. But okay, I can also change.

Reply:
@sleipnir Makes it easier to notice and fix inadvertent changes and keep things up to date on chanes. Thanks!

Reply:
Done!

Reply:
@sleipnir Looks good—docs-test CI fails, you might want to have a look here: https://travis-ci.com/github/cloudstateio/cloudstate/jobs/327793454#L348

Reply:
I've already seen that. The job running now is https://travis-ci.com/github/cloudstateio/cloudstate/builds/163811845